### PR TITLE
[stable/sysdig] Add 'How to upgrade to last version' to the README

### PR DESCRIPTION
**Pull request by tembleking**
#### What this PR does / why we need it:

This adds documentation about how to upgrade the Sysdig Agent to the latest version.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sysdig/README.md [hunk 1]
```diff
@@ -176,6 +176,32 @@ And then, upgrade Helm chart with:
 $ helm upgrade my-release -f values.yaml stable/sysdig
 ```
 
+## How to upgrade to the last version
+
+In case you deployed the chart with a values.yaml file, you just need to modify (or add if it's missing) the `agent.tag` field and execute:
```
Comment (bencer) [inline stable/sysdig/README.md]: this shouldn't be image.tag ?

File: stable/sysdig/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Sysdig Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v1.4.14
+
+### Minor changes
+
+* Add "How to upgrade to the last version" to the README.md
```
Comment (bencer) [inline stable/sysdig/CHANGELOG.md]: no need to bump version for doc changes

File: stable/sysdig/README.md [hunk 3]
```diff
@@ -176,6 +176,32 @@ And then, upgrade Helm chart with:
...
+
+```bash
+$ helm install --name my-release -f values.yaml stable/sysdig
+```
+
+If you deployed the chart setting the values as CLI parameters, like for example:
```
Comment (bencer) [inline stable/sysdig/README.md:188]: you say --name my-release and then --name sysdig which dont match.

can you explain if we need to use the same or different?

**Comment by zanhsieh** [comment]
@tembleking 
Could you bump the chart version to 1.14.6? Thank you.

**Pull request closed without merging**